### PR TITLE
Warn on broken links

### PR DESCRIPTION
**Pull request by Dzoge**

### CODE REVIEW

File: upload-docs/docusaurus/docusaurus.config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ module.exports = {
   tagline: 'Documentation validation',
   url: 'https://docs-test.private.e-conomic.ws',
   baseUrl: '/',
-  onBrokenLinks: 'throw',
+  onBrokenLinks: 'warn',
```
Comment (pkmetski) [inline upload-docs/docusaurus/docusaurus.config.js:6]: Isn't throwing good, though?
This action is all about documentation, and if the link is broken, this should warrant a throw, imo

Comment (pkmetski) [reply upload-docs/docusaurus/docusaurus.config.js:6]: Unless the throw is too late in the process...

**Review by pkmetski — APPROVED** [review]

File: upload-docs/docusaurus/docusaurus.config.js (hunk 1 above)
Comment (Dzoge) [reply upload-docs/docusaurus/docusaurus.config.js:6]: Yes it is. We can refactor this on the next update.

Comment (pkmetski) [reply upload-docs/docusaurus/docusaurus.config.js:6]: It is, as it is too late?

Comment (Dzoge) [reply upload-docs/docusaurus/docusaurus.config.js:6]: Sorry for that, yes, it is late at this time in some workflows. That can be fixed.

**Pull request merged**